data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/GenericExpressionEvaluator_ConditionalIT.java

```diff
@@ -94,7 +94,7 @@ void testConditionalExpressionEvaluator(final String expression, final Event eve
     void testGenericExpressionEvaluatorWithMultipleThreads(final String expression, final Event event, final Boolean expected) {
         final GenericExpressionEvaluator evaluator = applicationContext.getBean(GenericExpressionEvaluator.class);
 
-        final int numberOfThreads = 50;
+        final int numberOfThreads = 10;
```

Comment:
Why is this file being changed here? Why are the number of threads reduced?

Reply:
That test runs over 50 times, and it was spawning 50 threads for each test. My build failed every time due to that with an error about being out of threads